test/reduce/reduce_test_util.cpp

```diff
@@ -68,5 +70,28 @@ void NopDiagnostic(spv_message_level_t /*level*/, const char* /*source*/,
                    const spv_position_t& /*position*/,
                    const char* /*message*/) {}
 
+
```

Comment:
This is not used yet, but can be used when debugging tests and could be used in larger, more complex tests to ensure there is some information.